templates/Manifest.json

```diff
@@ -3,12 +3,12 @@
   "name": "chef-client-{{ platform }}{{ suffix }}",
   "publisher": "chef-software",
   "version": "{{ version }}",
-  "displayName": "Chef Extension ({{ version }}{{ extra }} {{ platform }})",
+  "displayName": "Chef Extension - {{ platform }}",
```

Comment:
Can we have `"Chef VM Extension for {{ platform }}"` please - as per the title at https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/chef

Reply:
Done now (I think)